packages/amplify-cli-core/src/types.ts

```diff
@@ -23,6 +23,7 @@ export type $TSContext = {
   migrationInfo: $TSAny;
   projectHasMobileHubResources: boolean;
   prompt: $TSAny;
+  patching: $TSAny;
```

Comment:
Meaningful change ^

Reply:
what does `patching` mean ?

Reply:
https://github.com/aws-amplify/amplify-cli/blob/608455b8f76fc2ac18d316ebb2cb901a15b58231/packages/amplify-cli-core/src/context/context-extensions.ts#L167

Reply:
I just updated `$TSContext` with a more meaningful type

Reply:
yeah. I saw that definition. I still have no idea what it does.
btw. should property be named patchingHandler or patcher ? it's kind of verb-ish for the property.

Reply:
It doesn't actually look like it's used anywhere in the application. So this can likely be marked as deprecated, then removed in the future. Agreed the name is weird, but I think renaming is out of scope of this PR.

Reply:
if that's dead code why not remove it now ?

Reply:
Fair enough. Removed.